app/retail/templates/emails/campaigns/email_campaign_day_1.html

```diff
@@ -66,24 +66,23 @@ <h1>Day 1: Growing Open Source</h1>
       {% blocktrans %}
       As time passes, we’ll send you more ways to get involved in Gitcoin. For now, we have a few areas you can nerd out with us, depending on where you’re most comfortable.
         <ul>
-            <li><a href="http://gitcoin.co/slack">Join the Slack channel</a> and say hi on #community-intros! </li>
-            <li><a href="http://twitter.com/gitcoin">Follow us on Twitter</a> for updates on open source software. </li>
-            <li><a href="https://www.youtube.com/channel/UCeKRqRjzSzq5yP-zUPwc6_w/featured">Add the Gitcoin Livestream to your cal (<a hred="https://calendar.google.com/calendar/ical/7rq7ga2oubv3tk93hk67agdv88%40group.calendar.google.com/public/basic.ics">iCal</a>, <a href="https://calendar.google.com/calendar/hosted/consensys.net/event?action=TEMPLATE&hl=en&text=Gitcoin%20Community%20Livestream&dates=20180427T160000%2F20180427T173000&location=https%3A%2F%2Fconsensys.zoom.us%2Fj%2F385625394&ctz=America%2FChicago&details">Google</a>) for Friday at 5PM EST. We’ll have two weekly product demo’s from the blockchain & open source ecosystems (like <a href="https://www.youtube.com/watch?v=E9c-6nlYyVo&pbjreload=10">Decentraland</a> and <a href="https://www.youtube.com/watch?v=EkZHTzGJMcY">Meta Transactions</a> with Austin Griffith. 
-            </li>
+            <li><a href="https://gitcoin.co/slack">Join the Slack channel</a> and say hi on #community-intros! </li>
+            <li><a href="https://twitter.com/gitcoin">Follow us on Twitter</a> for updates on open source software. </li>
+            <li><a href="https://www.youtube.com/channel/UCeKRqRjzSzq5yP-zUPwc6_w/featured">Add the Gitcoin Livestream</a> to your cal (<a href="https://calendar.google.com/calendar/ical/7rq7ga2oubv3tk93hk67agdv88%40group.calendar.google.com/public/basic.ics">iCal</a>, <a href="https://calendar.google.com/calendar/hosted/consensys.net/event?action=TEMPLATE&hl=en&text=Gitcoin%20Community%20Livestream&dates=20180427T160000%2F20180427T173000&location=https%3A%2F%2Fconsensys.zoom.us%2Fj%2F385625394&ctz=America%2FChicago&details">Google</a>) for Friday at 5PM EST. We’ll have two weekly product demo’s from the blockchain & open source ecosystems (like <a href="https://www.youtube.com/watch?v=E9c-6nlYyVo&pbjreload=10">Decentraland</a> and <a href="https://www.youtube.com/watch?v=EkZHTzGJMcY">Meta Transactions</a> with Austin Griffith.</li>
         </ul>
       {% endblocktrans %}
 
     </p>
 
     <p>
       {% blocktrans %}
-      Oh, and <a href="https://gitcoin.co/kudos/redeem/onboardingtree_01302019">here’s akudos</a> for getting to the bottom of this e-mail :) welcome to the Gitcoin Family! 
+      Oh, and <a href="https://gitcoin.co/kudos/redeem/onboardingtree_01302019">here’s akudos</a> for getting to the bottom of this e-mail :) welcome to the Gitcoin Family!
```

Comment:
since you are fixing a lot of typos and such here - can you fix the `akudos` to say `a kudos`? :)

Reply:
Haha, you right. Fixed.